projects/plugins/jetpack/modules/widgets/blog-stats.php

```diff
@@ -99,7 +111,7 @@ public function form( $instance ) {
 			<label for="<?php echo esc_attr( $this->get_field_id( 'hits' ) ); ?>"><?php esc_html_e( 'Pageview Description:', 'jetpack' ); ?></label>
 			<input class="widefat" id="<?php echo esc_attr( $this->get_field_id( 'hits' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'hits' ) ); ?>" type="text" value="<?php echo esc_attr( $instance['hits'] ); ?>" />
 		</p>
-		<p><?php esc_html_e( 'Hit counter is delayed by up to 60 seconds.', 'jetpack' ); ?></p>
+		<p><?php esc_html_e( 'Hit counter is delayed by up to 5 minutes.', 'jetpack' ); ?></p>
```

Comment:
The caching time has increased since the widget was released (I think it happened when moving functions to the Stats package)

https://github.com/Automattic/jetpack/blob/57221b8fc8d1dcd09b00d956b873c4464205eb8f/projects/packages/stats/src/class-wpcom-stats.php#L31-L36